tests/test_simulation.py

```diff
@@ -140,7 +141,7 @@ def testSimulationCached(self):
                 RadialCTFFilter(defocus=d) for d in np.linspace(1.5e4, 2.5e4, 7)
             ],
             seed=0,
-            noise_filter=IdentityFilter(),
+            noise_adder=CustomNoiseAdder(noise_filter=IdentityFilter()),
```

Comment:
What's the difference between giving simulation a ZeroFilter and an IdentityFilter?

Reply:
And is the default behavior sensible.. like do we want it to be completely clean images if no NoiseAdder is given

Reply:
At least three questions here, which I think could have been answered by reading and understanding the existing code, but I'll explain anyway.

> What's the difference between giving simulation a ZeroFilter and an IdentityFilter?

First one is zeros, other one is ones...

The filters are not the same.  Zero would actually be zero noise, ie, adding an array of zeros. This would be the same as not having a noise adder. 

Identity is going to run the code in `NoiseAdder._forward`, which will generate noise with a specific seed and scale it by `1**0.5=1`.  Changing this will break several unit tests (likely anything hardcoded relating to legacy sims...).  It isn't really the purpose of this PR.

> And is the default behavior sensible.. like do we want it to be completely clean images if no NoiseAdder is given

Simulation still reproduces the prior behavior (Simulation produces clean images by default). Yes I think that is sensible.  This test is setup with a specific noise array which is to match hard coded results.  This is less sensible, but not the purpose of this PR.

Now it may be possible to replace the custom thing with a white noise thing, which I will look into.

Reply:
To be totally fair, you were in the right to be confused by the additional code there.

Reply:
replaced with white noise var=1